style.css

```diff
@@ -1,6 +1,7 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
 
 /* @media screen and (max-width: 375px) { */
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.-

Reply:
Well  noted